src/Accounts/Authentication/Factories/AuthenticationFactory.cs

```diff
@@ -14,21 +14,22 @@
 
 using Hyak.Common;
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Identity.Client;
 using Microsoft.Rest;
 using System;
 using System.Linq;
 using System.Security;
 using Microsoft.Azure.Commands.Common.Authentication.Properties;
 using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Common.Authentication.Authentication.Clients;
 
 namespace Microsoft.Azure.Commands.Common.Authentication.Factories
 {
     public class AuthenticationFactory : IAuthenticationFactory
     {
         public const string AppServiceManagedIdentityFlag = "AppServiceManagedIdentityFlag";
 
-        public const string CommonAdTenant = "Common",
+        public const string CommonAdTenant = "organizations",
```

Comment:
Seems like we should have this constant in one place, and use it evwerywhere.  Also, same question as before about whether thsi works with MSA